unity-renderer/ProjectSettings/ProjectSettings.asset

```diff
@@ -574,13 +574,13 @@ PlayerSettings:
   webGLEmscriptenArgs: 
   webGLModulesDirectory: 
   webGLTemplate: PROJECT:decentraland
-  webGLAnalyzeBuildSize: 0
+  webGLAnalyzeBuildSize: 1
   webGLUseEmbeddedResources: 0
-  webGLCompressionFormat: 0
+  webGLCompressionFormat: 2
   webGLWasmArithmeticExceptions: 1
   webGLLinkerTarget: 1
   webGLThreadsSupport: 0
-  webGLDecompressionFallback: 1
+  webGLDecompressionFallback: 0
```

Comment:
Disabling the decompression fallback may produce problems when trying to run the build from `localhost` as the local servers may not provide the correct mime-type. @menduz did you confirm this is still working?

Reply:
I will need some help with that validation for `localhost`. For the initial tests, I changed the CDN and it disables streaming with a warning but also works correctly. Do you have the means to test it quicky to validate?